release/yum-repo/Earthfile

```diff
@@ -135,12 +122,13 @@ upload:
     ARG --required S3_BUCKET
     RUN --push \
         --mount type=secret,id=+secrets/earthly-technologies/release/keys/earthly-public.pgp,target=/release-key/earthly-public.pgp \
+        --secret MFA_ARN=+secrets/user/earthly-technologies/aws/mfa-arn \
+        --secret MFA_KEY=+secrets/user/earthly-technologies/aws/mfa-key \
         --mount type=secret,id=+secrets/user/earthly-technologies/aws/credentials,target=/root/.aws/credentials \
+        eval $(assume-developer-role 1) && \
```

Comment:
Why do we need to re-issue the assumed role credentials? They should be valid for an hour?

Reply:
I deliberately avoid keeping credentials in any layers, each `RUN` command must re-run `eval $(assume-developer-role 1)` to get fresh credentials.

Reply:
Fair enough; I wonder how many open sessions one is allowed though?

> I deliberately avoid keeping credentials in any layers

Out of curiosity, how is it different than keeping regular credentials on your machine when you aren't deploying the target image?